taskt/Core/Automation/Commands/RegexIsMatchCommand.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+using System.Xml.Serialization;
+using System.Data;
+using System.Windows.Forms;
+using System.Collections.Generic;
+using taskt.UI.Forms;
+using taskt.UI.CustomControls;
+using System.Text.RegularExpressions;
+
+namespace taskt.Core.Automation.Commands
+{
+    [Serializable]
+    [Attributes.ClassAttributes.Group("Regex Commands")]
+    [Attributes.ClassAttributes.Description("This command allows you to check if a match is found from text for given Regex Pattern")]
+    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to check if there is a match in text based on Regex Pattern")]
+    [Attributes.ClassAttributes.ImplementationDescription("This command implements IsMatch Action of Regex for given input Text and Regex Pattern and returns a boolean.")]
+    public class RegexIsMatchCommand : ScriptCommand
+    {
+        [XmlAttribute]
+        [Attributes.PropertyAttributes.PropertyDescription("Please input the data you want to perform regex on.")]
+        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [Attributes.PropertyAttributes.InputSpecification("Enter Variable or Text to apply Regex on.")]
+        [Attributes.PropertyAttributes.SampleUsage("**Hello** or **vSomeVariable**")]
+        [Attributes.PropertyAttributes.Remarks("")]
+        public string v_InputTextData { get; set; }
+
+        [XmlAttribute]
+        [Attributes.PropertyAttributes.PropertyDescription("Enter Regex Pattern.")]
+        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [Attributes.PropertyAttributes.InputSpecification("Enter a Regex Pattern to apply to check if a match exists.")]
+        [Attributes.PropertyAttributes.SampleUsage(@"^([\w\-]+)")]
+        [Attributes.PropertyAttributes.Remarks("")]
+        public string v_RegEx { get; set; }
+
+        [XmlAttribute]
+        [Attributes.PropertyAttributes.PropertyDescription("Please enter the variable to get Regex result.")]
+        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [Attributes.PropertyAttributes.InputSpecification("Enter a Variable to get result.")]
+        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
+        [Attributes.PropertyAttributes.Remarks("")]
+        public string v_ResultVariable { get; set; }
```

Comment:
Please take a look at GetDataRowValueCommand.cs for an example of what the output variable field should look like. Specifically the v_UserVariableName XmlAttribute and the way it's rendered. This is one of those inconsistencies that pops up in several commands that produce output variables, so we'll have to keep it in mind when we do the clean pass.

Reply:
These changes have been accommodated.